src/redux/accountSelectors.ts

```diff
@@ -70,6 +70,60 @@ export const getNetAPY = createSelector(
   },
 );
 
+export const getNetAPY_NEW = createSelector(
+  (state: RootState) => state.assets,
+  (state: RootState) => state.account,
+  (state: RootState) => state.app,
+  (assets, account, app) => {
+    if (!hasAssets(assets)) return 0;
+    const brrrTokenId = app.config.booster_token_id;
+    const sum = Object.keys(assets.data)
+      .filter((t) => t !== brrrTokenId)
```

Comment:
That's shouldn't be necessary, since META is somewhat similar. And eventually BRRR might have a price